content/_data/changelogs/lts.yml

```
@@ -3604,6 +3604,213 @@
     message: |-
       Users with extended read permission now get a more read-only looking UI.
 
+- version: "2.235.1"
+  date: 2020-06-17
+  lts_predecessor: "2.222.4"
+  lts_baseline: "2.235"
+
+  changes: # compared to lts_baseline 2.235
+
+  - type: major bug
+    category: regression
+    pull: 4713
+    issue: 62163
+    authors:
+    - daniel-beck
+    message: |-
+      Make plugin manager work on Internet Explorer 11 again (regression in 2.231).
+  - type: major bug
+    category: regression
+    pull: 4712
+    issue: 61920
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent telemetry warnings about missing <code>javax.annotation</code> classes when running with Java 11 (regression in 2.231).
+  - type: major bug
+    category: regression
+    pull: 4714
+    issue: 62181
+    authors:
+    - jglick
+    message: |-
+      Fix a deadlock involving custom loggers during agent startup (regression in 2.231).
+  - type: bug
+    category: bug
+    pull: 4718
+    issue: 62231
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent Old Data Monitor from failing plugin loading in the case of class field unmarshalling issues.
+
+  lts_changes: # compared to lts_predecessor 2.222.4
+
+  # Plugin manager improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4513
+    issue: 61166
+    references:
+    - issue: 61166
+    - pull: 4580
+    - pull: 4513
+    - pull: 4588
+    - pull: 4534
+    - pull: 4591
+    - pull: 4535
+    - pull: 4589
+    - pull: 4584
+    authors:
+    - daniel-beck
+    message: |-
+      Don't show all available plugins by default; use search field to find plugins.
+      Indicate when security issues would be addressed by an update in plugin manager.
+      Sort plugins by popularity on the <em>Available</em> plugin manager tab if the update site provides popularity data.
+      Show plugin categories as labels in the plugin manager instead of grouping them into different table sections.
+      Allow use of multiple space-separated filter terms in plugin manager.
+      Allow linking to plugin manager URLs with pre-filled filter field.
+      Link labels in the plugin manager to pre-filtered lists.
+      Show plugin release date in plugin manager.
+      List plugins that failed to load on the Installed tab of the plugin manager.
+      Highlight in the plugin manager when plugins are looking for new maintainers ("Adopt this plugin").
```

This one is a bit much IMO. How about this?

```suggestion
      Various improvements to the plugin manager, including:
      It no longer shows all available plugins by default; use search field to find plugins.
      They are now sorted by popularity by default.
      Additionally, categories are no longer used to group plugins, instead they're shown as labels.
```